src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,11 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    /**
+     * Returns whether the command will mutate the storage file
```

Comment:
Would be more precise if the header comment mentioned whether true or false is returned for mutating/non-mutating commands.